docs/templates/addon/index.rst

```diff
@@ -0,0 +1,22 @@
+=====
+addon
+=====
+
+.. topic:: Description
+
+    Creating an addon package and extending it with subtemplates.
+
+..  toctree::
```

Comment:
This will fail in all documentation tests we run :)

Since we had last year the same issue in the training docs we settled on only ONE toctree.